src/main/flight/mixer.c

```diff
@@ -659,7 +660,13 @@ static void applyFlipOverAfterCrashModeToMotors(void)
                 signPitch*currentMixer[i].pitch +
                 signRoll*currentMixer[i].roll +
                 signYaw*currentMixer[i].yaw;
-
+                
+            if (motorOutput < 0 && mixerConfig()->crashflip_motor_percent > 0)
+            {   
```

Comment:
Brace on the same line as the condition please, only functions use braces on a new line (K & R style).